src/interface/src/app/map/map-manager.ts

```diff
@@ -154,12 +154,8 @@ export class MapManager {
     this.setUpDrawingHandlers(map.instance!);
     this.setUpClickHandler(map, createDetailCardCallback);
 
-    // Since maps are synced, pan and zoom event handlers only need
```

Comment:
this was not really true, maps are synced when several are shown, but this breaks if you are showing only one map and selected map 3 for example.